src/components/charts/PercentageStackedBarChart/index.jsx

```diff
@@ -122,6 +130,8 @@ PercentageStackedBarChart.defaultProps = {
     fontWeight: 600,
     position: 'center',
   },
+  lockValue: -1,
+  lockMessage: 'You cannot see this chart because it contains cohort under 1000 subjects',
```

Comment:
I would rephrase this to something like 'This chart is hidden because it contains a cohort with fewer than 1000 subjects'